### PR TITLE
updating readme w/ zftp snippet

### PR DESCRIPTION
**Pull request by anaxceron**
Added snippet to Step 6 of [Create a Team Configuration File](https://github.com/zowe/vscode-extension-for-zowe/tree/main/packages/zowe-explorer-ftp-extension#create-a-team-configuration-file).

This resolves [Issue 1899](https://github.com/zowe/vscode-extension-for-zowe/issues/1899)

### CODE REVIEW

File: packages/zowe-explorer-ftp-extension/README.md [hunk 1]
```diff
@@ -54,7 +54,15 @@ If you do not have an existing Zowe FTP profile, you can create one graphically
 3. Click the **+** icon.
 4. Select **Create a New Team Configuration File**.
 5. If no workspace is opened a global configuration file will be created. If a workspace is opened, chose either a global configuration file or a project-level configuration file.
-6. Edit the config file to include the host and other connection information.
+6. Edit the config file to include the host and other connection information:
+   ```
+   "zftp": {
+         "type": "zftp",
+         "properties": {YOURHOSTNAME},
```
Comment (t1m0thyj) [inline packages/zowe-explorer-ftp-extension/README.md:61]: Should we expand this to include a "host" property?
```
"properties": {
  "host": "YOURHOSTNAME"
},
```

**Comment by JillieBeanSim** [comment]
@anaxceron do we want this included in 2.8.1 release?

**Review by t1m0thyj — APPROVED** [review]

**Comment by JillieBeanSim** [comment]
@t1m0thyj @zFernand0 I see this PR has the 2.8.1 milestone but is based against the main branch

**Comment by JillieBeanSim** [comment]
with it being approved we could merge and cherry pick the commit for maintenance branch

**Pull request merged**